src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -63,3 +63,74 @@ def __init__(self):
 
 
 RUNTIME_STACKS = os.path.abspath(os.path.join(os.path.abspath(__file__), '../resources/WebappRuntimeStacks.json'))
+
+APP_NAME_ADJECTIVES = [
```

Comment:
Nit: Can we add these to a JSON file as well & reference the file here?

Reply:
@calvinsID remove the test yaml file with the exposed secrets & re-record. https://github.com/Azure/azure-cli/pull/16267/files#diff-64d4540d613959d0ba1fb29e8591c6fa5612d2e3bb6bee1513d45949b4bc6672

Reply:
Suppressed credential scanner for the webapp up test since its not a real secret